types/storybook__polymer/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```

Comment:
There shouldn’t be anything in `rules`

Reply:
Don't know if this is documented anywhere, but there are a large number of packages with tslint rules. I removed them though

Reply:
It’s here: https://github.com/DefinitelyTyped/DefinitelyTyped#lint

At the time we started linting DefinitelyTyped, there were already thousands of packages. Fixing them all to conform would have been a herculean effort with a questionable rate of return, so we auto-generated tslint configs disabling the smallest number of rules possible for each package. That‘s why you see existing exceptions—in a perfect world, there would be none.